src/components/ErrorComponent.tsx

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable max-len */
+import React from 'react';
+import { ErrorMessages } from './PeoplePage';
+
+interface ErrrorComponentProps {
```

Comment:
There is a typo in the interface name `ErrrorComponentProps`. It should be corrected to `ErrorComponentProps` to maintain consistency and avoid confusion.